deps/rabbitmq_management/selenium/test/oauth/with-idp-initiated/redirection-after-login.js

```diff
@@ -0,0 +1,46 @@
+const { By, Key, until, Builder } = require('selenium-webdriver')
+require('chromedriver')
+const assert = require('assert')
+const { buildDriver, goToExchanges, captureScreensFor, teardown, goToHome } = require('../../utils')
+
+const SSOHomePage = require('../../pageobjects/SSOHomePage')
+const UAALoginPage = require('../../pageobjects/UAALoginPage')
+const ExchangesPage = require('../../pageobjects/ExchangesPage')
+const OverviewPage = require('../../pageobjects/OverviewPage')
```

Comment:
It is cosmetic, but given that Overview is not used in this test , we do not need to declare it,

Reply:
Oh sorry, I was creating a testing using the overview page, but I changed it later and forgot to remove here, thanks